masterlist.yaml

```
@@ -5516,12 +5516,30 @@ plugins:
   - name: 'TTWLODGen.esp'
     url: [ 'https://taleoftwowastelands.com/content/fnvlodgen' ]
     group: *lodGroup
+
   - name: 'tmzLODadditions.esp'
-    url: [ 'https://www.nexusmods.com/newvegas/mods/58562' ]
+    - link: 'https://www.nexusmods.com/newvegas/mods/61206/'
+      name: 'LOD additions and improvements'
```

Url format needs to be fixed.
```suggestion
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/61206/'
        name: 'LOD additions and improvements'
```